cobra/flux_analysis/summary.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 
+"""Define the Summary class."""
+
 from __future__ import absolute_import, division
 
```

Comment:
Not sure this should be located in `flux_analysis`. The summary class provides information on the core objects, not necessary part of the flux_analysis.

Reply:
I wasn't sure on this as well so I just let it be there. I was hoping we could discuss and come up a proper place for this. I also have another concern: shouldn't be the subclasses implemented in separate modules? I think that would increase the maintainability.

Reply:
@Midnighter @cdiener Shall we shift this to core as @matthiaskoenig suggests? Also, any thoughts on the splitting of subclasses?